core/src/main/java/io/aiven/klaw/repository/TopicRequestsRepo.java

```diff
@@ -96,4 +106,12 @@ Long countRequestorsTopicRequestsGroupByStatusType(
       @Param("topicStatus") String topicStatus);
 
   void deleteByTenantId(int tenantId);
+
+  private Map<String, Long> toStringLongMap(List<Object[]> list) {
+    var result = new HashMap<String, Long>(list.size());
```

Comment:
@snuyanzin would be nice if the variable names match the context, like topicRequests etc

Reply:
done